qc_grader/challenges/qdc_2025/__init__.py

```diff
@@ -4,5 +4,21 @@
     grade_lab1_ex2,
     grade_lab1_ex3,
     grade_lab1_ex4,
-    grade_lab1_ex5
+    grade_lab1_ex5,
+)
+
+from .qdc25_lab2 import (
+    submit_name,
```

Comment:
You can remove this `submit_name` import - once it's imported for one lab in `__init__.py` like [this](https://github.com/qiskit-community/Quantum-Challenge-Grader/blob/74531b900d53bc8a9373e37aeddc665ce4fb0314/qc_grader/challenges/test_challenge/__init__.py#L15), you don't need to import it anymore